Sources/PointFree/About.swift

```diff
@@ -8,32 +8,149 @@ import Prelude
+let aboutResponse: Middleware<StatusLineOpen, ResponseEnded, Tuple3<Database.User?, Stripe.Subscription.Status?, Route?>, Data> =
   writeStatus(.ok)
     >-> map(lower)
     >>> respond(
       view: aboutView,
-      layoutData: { currentUser in
+      layoutData: { currentUser, subscriptionStatus, currentRoute in
         SimplePageLayoutData(
+          currentRoute: currentRoute,
+          currentSubscriptionStatus: subscriptionStatus,
           currentUser: currentUser,
           data: unit,
-          title: "About Us"
+          title: "About"
         )
     }
 )
 
 private let aboutView = View<Prelude.Unit> { _ in
   gridRow([
-    gridColumn(sizes: [.mobile: 12], [
-      div([`class`([Class.padding([.mobile: [.all: 4]])])], [
-
-        h1([`class`([Class.h1])], ["About Us"]),
-        p(["Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit."]),
-        p(["Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit."]),
-        p(["Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit."]),
-        p(["Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit."]),
-        p(["Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit."])
-        ])
+    gridColumn(sizes: [.mobile: 12, .desktop: 9], [style(margin(leftRight: .auto))], [
+      div([`class`([Class.padding([.mobile: [.all: 3], .desktop: [.all: 4]])])],
+          aboutSectionView.view(unit)
+            + openSourceSection.view(unit)
+            + hostsSection.view(unit)
+      )
       ])
     ])
 }
+
+let aboutSectionView = View<Prelude.Unit> { _ in
```

Comment:
after I merge this feel free to branch off and thrash the copy any way you want!